### ISSUE TITLE
ElementNotFoundError: faac

### ISSUE THREAD

**Issue by ahow628** [danielgtaylor/arista#144]
I do the following at the CLI:

arista-transcode -p H.264 "input file.mkv"

And get this error:

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/arista/transcoder.py", line 228, in _got_info
    self._setup_pass()
  File "/usr/lib/python2.7/dist-packages/arista/transcoder.py", line 542, in _setup_pass
    "aencoder")
gst.ElementNotFoundError: faac

It just hangs at that point and requires Ctrl+C. Any idea what is going on? I have all the gstreamers installed from the Ubuntu Software Center plus I did apt-get for the 'bad' and 'ugly' packages as well.

This is on a headless Mythbuntu 12.04 with XFCE desktop. Using arista-gtk just crashes.

Using the same CLI as above on my laptop with Ubuntu 12.04 works and so does the GTK version. I'd prefer to run this on the desktop and not on the laptop.

Thanks,
Andy

**Comment by ahow628** [comment]
Alright, the solution was pretty easy, but kind of strange nonetheless.

I went into Ubuntu Software Center on my desktop and went to the additional information screen. About halfway down the page is an "Addons" section. The part for faac was unchecked so I checked it.

On my laptop, that box was checked already, hence it could do the encode while the desktop could not. Not sure why there was a difference between the two Addon settings.
